tests/bluetooth/tester/src/btp/btp_mesh.h

```diff
@@ -1060,6 +1060,9 @@ struct btp_priv_node_id_set_cmd {
 } __packed;
 
 #define BTP_MESH_PROXY_PRIVATE_IDENTITY		0x72
+struct btp_proxy_priv_identity_cmd {
+	bool enabled;
```

Comment:
I'd suggest to use uint8_t here  (to be very strict C spec doesn't define sizeof bool)